### PR TITLE
superknowa application oepnsource

### PR DESCRIPTION
**Pull request by hitalukder**

### CODE REVIEW

File: 8. Application/superknowa-app/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Copy the package.json and package-lock.json files to the container
+COPY package*.json ./
+
+# Install the dependencies
+RUN npm install
+
+# Copy the rest of the application files to the container
+COPY ./src ./src
+COPY ./public ./public
+COPY ./.env ./.env
```
Comment (kevinxhuang) [inline 8. Application/superknowa-app/Dockerfile:16]: @hitalukder avoid adding .env if it has credentials in it.

File: 8. Application/superknowa-app/backend-openapi-document.json [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+                "required": [
+                    "loc",
+                    "msg",
+                    "type"
+                ],
+                "title": "ValidationError"
+            }
+        }
+    }
+}
```
Comment (kevinxhuang) [inline 8. Application/superknowa-app/backend-openapi-document.json:374]: @hitalukder add a new line here.

Comment (kevinxhuang) [reply 8. Application/superknowa-app/backend-openapi-document.json:374]: Same applies to a number of files.

File: 8. Application/superknowa-app/Dockerfile (hunk 1 above)
Comment (hitalukder) [reply 8. Application/superknowa-app/Dockerfile]: Good point. Removed the  line .env copy

File: 9. AI Alignment Tool/superknowa-app/src/components/AnswerFeedbackDialog.jsx [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+import { useTheme } from "@mui/material";
+import ThumbUpOutlinedIcon from "@mui/icons-material/ThumbUpOutlined";
+import ThumbDownOutlinedIcon from "@mui/icons-material/ThumbDownOutlined";
+import ThumbUpIcon from "@mui/icons-material/ThumbUp";
+import ThumbDownIcon from "@mui/icons-material/ThumbDown";
+import TextField from "@mui/material/TextField";
+import { additionalFeedback } from "../api/feedback/feedback_api";
+import { Box, Grid, Typography } from "@mui/material";
+
+const feedbackQustions = [
```
Comment (AnandDas227) [inline 9. AI Alignment Tool/superknowa-app/src/components/AnswerFeedbackDialog.jsx:27]: fix feedbackQuestions var name

File: 9. AI Alignment Tool/superknowa-app/src/components/global/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Footer() {
+  return (
+    <footer style={{ position: "absolute", bottom: 15, height: "40px", fontSize: 11, left: '400px', zIndex: 1 }}>
+      <div className="container">
+        <p style={{ margin: "20px" }}>Disclaimer - Please note that this content is made available to foster Embedded AI technology adoption. The content may include systems & methods pending patent with the USPTO and protected under US Patent Laws. Copyright - 2023 IBM Corporation. In case of any questions or support, please reach out to <a href = "mailto: kunal@ibm.com">kunal@ibm.com</a></p>
```
Comment (AnandDas227) [inline 9. AI Alignment Tool/superknowa-app/src/components/global/Footer.jsx:7]: 2024

File: 9. AI Alignment Tool/superknowa-app/src/components/global/FooterV1.jsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function FooterV1() {
+  return (
+    <footer style={{ height: "40px", fontSize: 11, zIndex: 1 }}>
+      <div style={{display: 'flex', justifyContent: 'center'}} >
+        <p style={{ margin: "20px" }}>Please note that this content is made available to foster Embedded AI technology adoption. The content may include systems & methods pending patent with USPTO and protected under US Patent Laws. Copyright - 2023 IBM Corporation</p>
```
Comment (AnandDas227) [inline 9. AI Alignment Tool/superknowa-app/src/components/global/FooterV1.jsx:7]: 2024

File: 9. AI Alignment Tool/superknowa-app/src/components/multimodel/MultiAnswerFeedbackDialog.jsx [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+import { useTheme } from "@mui/material";
+import ThumbUpOutlinedIcon from "@mui/icons-material/ThumbUpOutlined";
+import ThumbDownOutlinedIcon from "@mui/icons-material/ThumbDownOutlined";
+import ThumbUpIcon from "@mui/icons-material/ThumbUp";
+import ThumbDownIcon from "@mui/icons-material/ThumbDown";
+import TextField from "@mui/material/TextField";
+import { additionalFeedback, multiAdditionalFeedback } from "../../api/feedback/feedback_api";
+import { Box, Grid, Typography } from "@mui/material";
+
+const feedbackQustions = [
```
Comment (AnandDas227) [inline 9. AI Alignment Tool/superknowa-app/src/components/multimodel/MultiAnswerFeedbackDialog.jsx:23]: fix var name

File: 9. AI Alignment Tool/superknowa-app/src/scenes/Login.jsx [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+            justifyContent={"center"}
+            justifyItems={"center"}
+            mt={'20px'}
+          >
+            <Button
+              sx={{ width: 250, height: 40, textTransform: 'capitalize' }}
+              size="large"
+              variant="outlined"
+              component={Link}
+              href="https://www.ibm.com/account/reg/us-en/signup?formid=urx-19776&target=https%3A%2F%2Flogin.ibm.com%2Foidc%2Fendpoint%2Fdefault%2Fauthorize%3FqsId%3Df6c9d92c-37e6-4c93-9ba3-ec3b89cc403f%26client_id%3DOGQzYWY3MWQtZjc0YS00"
```
Comment (AnandDas227) [inline 9. AI Alignment Tool/superknowa-app/src/scenes/Login.jsx:89]: redirects to target page then breaks

**Review by Shivam6693 — APPROVED** [review]
- Readme reviewed
- Code review completed by Kevin and Anand

**Pull request merged**